### PR TITLE
Move logging to ephemeral store (where applicable)

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

File: test/integration/cloud-default/serverspec/default_spec.rb [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    describe file '/etc/cassandra/cassandra-rackdc.properties' do
+      it { is_expected.to be_file }
+      describe '#content' do
+        subject { super().content }
+        it { is_expected.to include 'dc=us-east-1' }
+        it { is_expected.to include 'rack=b' }
+      end
+    end
+  end
+
```
Comment (eherot) [inline test/integration/cloud-default/serverspec/default_spec.rb]: Below is a section I added (similar additions were made to all of the other default_spec files).

File: recipes/opscenter.rb [hunk 2]
```diff
@@ -5,6 +5,22 @@
...
+include_recipe 'storage'
+
+if node['storage']['ephemeral_mounts']
+  node.set['et_cassandra']['opscenter']['config']['logging']['log_path'] =
+    "#{node['storage']['ephemeral_mounts'].first}/opscenter/opscenterd.log"
+  node.default['et_cassandra']['datastax-agent']['log_dir'] =
+    "#{node['storage']['ephemeral_mounts'].first}/datastax-agent"
+  directory node.default['et_cassandra']['datastax-agent']['log_dir'] do
+    owner     node['et_cassandra']['user']
+    group     node['et_cassandra']['user']
```
Comment (jeffbyrnes) [inline recipes/opscenter.rb]: Isn’t there a `group` attribute?

File: recipes/opscenter.rb [hunk 3]
```diff
@@ -5,6 +5,22 @@
...
 
 include_recipe 'et_cassandra::repo'
+include_recipe 'storage'
+
+if node['storage']['ephemeral_mounts']
+  node.set['et_cassandra']['opscenter']['config']['logging']['log_path'] =
+    "#{node['storage']['ephemeral_mounts'].first}/opscenter/opscenterd.log"
+  node.default['et_cassandra']['datastax-agent']['log_dir'] =
+    "#{node['storage']['ephemeral_mounts'].first}/datastax-agent"
+  directory node.default['et_cassandra']['datastax-agent']['log_dir'] do
```
Comment (jeffbyrnes) [inline recipes/opscenter.rb]: Why the mix & match of attribute precedence? Why not use `set` throughout?

File: recipes/opscenter.rb [hunk 4]
```diff
@@ -103,3 +119,18 @@
...
+  owner  'root'
+  group  'root'
+  mode   0644
+end
+
+template "#{agent_conf_path}/log4j.properties" do
+  source 'datastax-log4j.properties.erb'
+  owner  'root'
+  group  'root'
+  mode   0644
```
Comment (jeffbyrnes) [inline recipes/opscenter.rb]: No need to specify these defaults, though it’s not harmful either.

**Comment by jeffbyrnes** [comment]
@eherot just some nits, but generally :+1:

**Pull request merged**